modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts

```diff
@@ -147,7 +147,8 @@ const get = function (dom: DOMUtils) {
 
     strikethrough: [
       { inline: 'span', styles: { textDecoration: 'line-through' }, exact: true },
-      { inline: 'strike', remove: 'all', preserve_attributes: [ 'class', 'style' ] }
+      { inline: 'strike', remove: 'all', preserve_attributes: [ 'class', 'style' ] },
+      { inline: 's', remove: 'all', preserve_attributes: [ 'class', 'style' ] }
```

Comment:
If we are adding this, [Legacy Output](https://github.com/tinymce/tinymce/blob/develop/modules%2Ftinymce%2Fsrc%2Fplugins%2Flegacyoutput%2Fmain%2Fts%2Fcore%2FFormats.ts#L40) will have to be updated as well.

Reply:
@ltrouton I'm not sure about that... it should only list legacy HTML elements, so should `s` be in there?

Reply:
new PR targeting master: https://github.com/tinymce/tinymce/pull/6260